recipes/corrade/all/conanfile.py

```diff
@@ -65,6 +48,7 @@ def configure(self):
         if self.settings.compiler == "Visual Studio" and tools.Version(self.settings.compiler.version.value) < 14:
             raise ConanInvalidConfiguration("Corrade requires Visual Studio version 14 or greater")
         if tools.cross_building(self.settings):
```

Comment:
```suggestion
        if tools.cross_building(self):
```

Reply:
This is the recommend way now? Good to know :smile:

Reply:
yes, this was done in order to support the new cross-building model